### DISCUSSION TITLE
Transmission with Flood on Argo Tunnel

### DISCUSSION THREAD

**Discussion by j3ws3r** [jesec/flood#373] [Q&A]
Hello, 

So I've set up as I mentioned - Transmission with Flood running through Cloudflare Argo Tunnel. 
But I encountered a strange problem which I can't resolve at all. 
When I access flood locally - everything runs smooth and fast. 
When I access flood through Argo Tunnel - Everything kinda works, login loads fast, but then it takes considerable amount of time to load the main page with torrents. It takes longer to upload files and after each refresh that loading time still is the same.. 
Other web services which are running on tunnel works really fast - no noticeable difference between tunnel or local. 
Other services: Cockpit, Netstats, a couple of my own web projects. 

Could someone help a bit here? Or maybe someone tried something like that?

**Comment by jesec** [comment]
It is a very unique case. 

FYI: Flood mostly uses Server-Sent Events for backend-frontend communication. 

You may be able to find more info and workaround with that in mind. 

At the moment, I don't think there is anything I can do unfortunately.

**Reply by j3ws3r** [reply]
So I played with server configuration a bit and found out that it works normally when I set up reverse proxy through nginx(apache also works) and just give a subdirectory to flood and not making any additional rules in Cloudflare's Argo Tunnel config file. 
Hope that helps if someone will face similar issue.
